### ISSUE TITLE
Make progress bar on Termux work like Windows

### ISSUE THREAD

**Issue by WyntrHeart** [upx/upx#826]
This issue tracker is **ONLY** used for requesting sensible features.
Please use [stackoverflow](https://stackoverflow.com) for supporting issues.

## What's the problem (or question)?

On Windows the progrss bar of UPX is printed on a single line in the terminal. On Android in Termux UPX prints a new line to the console each time the progress bar updates, which appears to be many times per second. Hundreds of lines are being printed.

## What should have happened?

It would be nice if the progress bar on Android only printed a single line, and overwrites that line when it updates, the same as it does on Windows.

## Do you have an idea for a solution?

N/A

## How can we reproduce the issue?

N/A
(You know, this template doesn't really make much sense when I selected the option to suggest a feature, not report a bug)

## Please tell us details about your environment.

* UPX version used (`upx --version`): 4.2.4
* Host Operating System and version: Android 9
* Host CPU architecture: Aarch64
* Target Operating System and version: Android 9
* Target CPU architecture: Aarch64

**Comment by markus-oberhumer** [comment]
Indeed, the current console code has problems if you have few `$COLUMNS`, and would benefit from an update to use more modern terminal features.

Workaround for now: `export UPX="--no-progress"`
